### DISCUSSION TITLE
Dockerfile misconfiguration not shown in https://avd.aquasec.com/misconfig?

### DISCUSSION THREAD

**Discussion by htmslh** [aquasecurity/trivy#7956] [Documentation]
### Description

When I access the usual misconfig database https://avd.aquasec.com/misconfig, usually there is the dockerfile option. Now it does not appear. Accessing the previous link directly still works. Is this a bug or are they going to be deprecated?

### Link

Misconfiguration Database: https://avd.aquasec.com/misconfig
Dockerfile misconfiguration: https://avd.aquasec.com/misconfig/dockerfile/

### Suggestions

Include the dockerfile option in https://avd.aquasec.com/misconfig

**Comment by nikpivkin** [comment]
Hi @hutomosaleh !

I'm investigating it.

**Reply by simar7** [reply]
@nikpivkin it is strange because https://avd.aquasec.com/misconfig/dockerfile/ and https://avd.aquasec.com/misconfig/dockerfile/general/ still resolve.

**Reply by nikpivkin** [reply]
Opened a [PR](https://github.com/aquasecurity/avd-generator/pull/104) with a fix for this.

**Comment by nikpivkin** [comment]
Track https://github.com/aquasecurity/avd-generator/issues/105
